docs/specification.md

```diff
@@ -13,41 +13,42 @@ Currently, `geff` supports zarr specifications [2](https://zarr-specs.readthedoc
+The `nodes\ids` array is a 1D array of node IDs of length `N` >= 0, where `N` is the number of nodes in the graph. Node ids must be unique. Node IDs can have any type supported by zarr (except floats), but we recommend integer dtypes. For large graphs, `uint64` might be necessary to provide enough range for every node to have a unique ID. In the minimal case of an empty graph, the `ids` array will be present but empty. 
+
 
 ### The `props` group and `node property` groups
-The `nodes\props` group will contain one or more `node property` groups, each with a `values` array and an optional `missing` array.
+The `nodes\props` group is optional and will contain one or more `node property` groups, each with a `values` array and an optional `missing` array.
 
 - `values` arrays can be any zarr supported dtype, and can be N-dimensional. The first dimension of the `values` array must have the same length as the node `ids` array, such that each row of the property `values` array stores the property for the node at that index in the ids array.
-- The `missing` array is an optional, a one dimensional boolean array to support properties that are not present on all nodes. A 1 at an index in the `missing` array indicates that the `value` of that property for the node at that index is None, and the value in the `values` array at that index should be ignored. If the `missing` array is not present, that means that all nodes have values for the property.
+- The `missing` array is an optional, a one dimensional boolean array to support properties that are not present on all nodes. A `1` at an index in the `missing` array indicates that the `value` of that property for the node at that index is None, and the value in the `values` array at that index should be ignored. If the `missing` array is not present, that means that all nodes have values for the property.
 
-!!! note
-    When writing a graph with missing properties to the geff format, you must fill in a dummy value in the `values` array for the nodes that are missing the property, in order to keep the indices aligned with the node ids.
+-  Geff provides special support for spatio-temporal properties, although they are not required. When `axes` are be specified in the `geff` metadata, each axis name identifies a spatio-temporal property. Spatio-temporal properties are not allowed to have missing arrays. Otherwise, they are identical to other properties from a storage specification perspective.
 
-- The `position` group is a special node property group that must be present if a `position_prop` is set in the `geff` metadata and does not allow missing properties.
 - The `seg_id` group is an optional, special node property group that stores the segmenatation label for each node. The `seg_id` values do not need to be unique, in case labels are repeated between time points. If the `seg_id` group is not present, it is assumed that the graph is not associated with a segmentation. 
```

Comment:
```suggestion
- The `seg_id` property is an optional, special node property that stores the segmenatation label for each node. The `seg_id` values do not need to be unique, in case labels are repeated between time points. If the `seg_id` property is not present, it is assumed that the graph is not associated with a segmentation. 
```